blueflood-core/src/integration-test/java/com/rackspacecloud/blueflood/service/ScheduleContextIntegrationTest.java

```diff
@@ -1,76 +0,0 @@
-/*
```

Comment:
There is also a unit test called `ScheduledContextTest`. This integration test only tests the stuff related to ZK.